extension/core_functions/scalar/list/list_reduce.cpp

```diff
@@ -263,8 +262,46 @@ static unique_ptr<FunctionData> ListReduceBind(ClientContext &context, ScalarFun
 	                                     has_initial);
 }
 
-static LogicalType ListReduceBindLambda(const idx_t parameter_idx, const LogicalType &list_child_type) {
-	return LambdaFunctions::BindTernaryLambda(parameter_idx, list_child_type);
+
+LogicalType LambdaFunctions::BindReduce(const idx_t parameter_idx, ClientContext &context,
```

Comment:
Maybe we can change the name here - its a bit confusing since we also have `ListReduceBind`.